modules/home-manager/newsboat.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, ... }:
+let
+  inherit (config.catppuccin) sources;
+
+  cfg = config.programs.newsboat.catppuccin;
+  enable = cfg.enable && config.programs.newsboat.enable;
+  theme = "themes/" + (if cfg.flavor == "latte" then "latte" else "dark");
+in
+{
+  options.programs.newsboat.catppuccin = lib.ctp.mkCatppuccinOpt "newsboat";
+
+  config = lib.mkIf enable {
+    xdg.configFile."newsboat/${theme}".source = "${sources.newsboat}/${theme}";
```

Comment:
do we need to link the file with `xdg.configFile` if the file can be included straight into newsboat from line 15?